client/js/components/statement/publicStatementModal/StatementModal.vue

```diff
@@ -1464,7 +1452,6 @@ export default {
 
       const postValidation = () => {
         if (this.dpValidate.statementForm === false) {
-          this.$nextTick(() => document.getElementById('statementFormErrors').focus())
```

Comment:
Why do you remove this behavior?

Reply:
I'm glad you ask, because I did not understand why we implemented this behavior. It did not seem useful or necessary to me, but maybe you know the reason?

Reply:
in a former initiative to make the statement flow accessible, we found an issue with how errors are handled for screen reader users. after submitting statements (when using screen reader), the user should be informed that, and which, errors prevent the statement from being submitted. this was implemented by setting focus on the error message which would then be read out. See [relevant commit](https://github.com/demos-europe/demosplan/commit/d3ea09c31f51cf53bc0630769671a44bb16de4ad) and [PR](https://github.com/demos-europe/demosplan/pull/2977).

Reply:
But can't we accomplish the same with `aria-live="assertive"`?

Reply:
i'm not sure. while the content would indeed be read out if the element changes, the focus is not moved to the error message. how does focus behave when errors are found?

Reply:
ok, then maybe i restore the previous implementation to be sure?

Reply:
Done in 03c3844179e7f2ac89890476944dd36773793fbd